packages/resource-database-helpers/src/dashboard/dashboard.types.ts

```diff
@@ -0,0 +1,37 @@
+export type DashboardId = string;
+export type DashboardName = string;
+export type DashboardDescription = string;
+
+export type DashboardSummary = {
+    id: DashboardId,
+    name: DashboardName,
+    description: DashboardDescription,
+    creationDate: Date,
+    lastUpdateDate: Date,
+};
+
+export type DashboardWidgetTitle = string;
+
+export enum DashboardWidgetType {
+  Line = 'line',
+  Scatter = 'scatter',
+  Bar = 'bar',
+  StatusGrid = 'status-grid',
+  StatusTimeline = 'status-timeline',
+  Kpi = 'kpi',
+  Table = 'table',
+}
+
+export type DashboardWidget = {
```

Comment:
This is the widget definition https://github.com/awslabs/iot-app-kit/blob/feature-dashboard-alpha/packages/dashboard/src/types.ts#L36